### PR TITLE
make selection count user friendly

### PR DESCRIPTION
**Pull request by mertkahyaoglu**
ss

Changes;
* added line count
* added tooltip
* added config option
* added specs
* edited readme.md & package.json

### CODE REVIEW

**Comment by mertkahyaoglu** [comment]
@kevinsawicki I used ```getRowCount``` instead of splitting as you said. 

I messed up the other PR and had to close it. I'm sorry about it

File: lib/selection-count-view.coffee [hunk 1]
```diff
@@ -1,15 +1,28 @@
...
     @classList.add('selection-count', 'inline-block')
 
+    @formatString = atom.config.get('status-bar.selectionCountFormat') ? '(%L, %C)'
     @activeItemSubscription = atom.workspace.onDidChangeActivePaneItem =>
       @subscribeToActiveTextEditor()
 
+    @subscribeToConfig()
     @subscribeToActiveTextEditor()
 
+    toolTipDisposable: null
```
Comment (kevinsawicki) [inline lib/selection-count-view.coffee]: This should be above the `initialize: ->` function.

The currently location has the `initialize` method returning a `{toolTipDisposable: null}` object

Comment (mertkahyaoglu) [inline lib/selection-count-view.coffee]: :+1:

File: lib/selection-count-view.coffee [hunk 2]
```diff
@@ -1,15 +1,28 @@
+_ = require 'underscore-plus'
+
 class SelectionCountView extends HTMLElement
+  toolTipDisposable: null
```
Comment (kevinsawicki) [inline lib/selection-count-view.coffee]: Actually, sorry, after thinking about this more I don't think it should be declared at like this unless the other disposables are as well like `configSubscription`.

So for consistency, I'd say either all or none, and for initial simplicity I'd say just none of them.

Comment (mertkahyaoglu) [inline lib/selection-count-view.coffee]: So, should I delete the line ?

Comment (kevinsawicki) [inline lib/selection-count-view.coffee]: Yeah, thanks.

Comment (mertkahyaoglu) [inline lib/selection-count-view.coffee]: Actually, I stole some lines from [here](https://github.com/atom/deprecation-cop/blob/master/lib/deprecation-cop-status-bar-view.coffee#L16). Should we also delete here ?

Comment (kevinsawicki) [inline lib/selection-count-view.coffee]: Yeah, that could/should be cleaned up eventually, although it isn't doing any harm, just more of a style/consistency issue.

**Comment by kevinsawicki** [comment]
This is a really nice addition, thanks for adding it :+1: 

This will be included in the next Atom release, 1.0.8

**Comment by mertkahyaoglu** [comment]
@kevinsawicki @benogle Thanks for your help. Learned a lot :sparkles:

**Pull request merged**